client/question/question.js

```diff
@@ -334,6 +334,20 @@ tie.constant('WRONG_LANGUAGE_ERRORS', {
     ],
     allowMultiline: false
   }, {
+    // Used the wrong casing for "True" or "False".
+    errorName: 'booleans',
+    regExString: '\\b(true|TRUE|false|FALSE)\\b',
+    feedbackParagraphs: [
+      {
+        type: 'text',
+        content: [
+          "Note that, in Python, we write boolean values as True and False. ",
```

Comment:
"Python uses True and False for boolean values, rather than true and false."

Reply:
Actually, I can't quite write that, since they may have written TRUE/FALSE instead, and it seems a bit clumsy to list all the possibilities. I'd also rather not break this out into more separate tests than necessary.

Do you feel strongly / have alternative wording?

Reply:
I'd just go with "Python only allows True and False for boolean values." or something.

Reply:
Done.